src/Schedule.h

```diff
@@ -624,6 +624,9 @@ class FuncSchedule {
     bool &async();
     bool async() const;
 
+    Expr &ring_buffer();
+    Expr ring_buffer() const;
```

Comment:
I believe the signature is supposed to be `const Expr &ring_buffer() const;`?

Reply:
Hm, good question -- not sure if it matters much in this case?

Reply:
I added "&" here, but other functions in that file are really inconsistent.